util/omrutil/gettimebase.c

```diff
@@ -92,6 +92,8 @@ getTimebase(void)
 	int64_t tsc_int;
 	asm volatile("mrs %0, cntvct_el0" : "=r"(tsc_int));
 	tsc = (uint64_t)tsc_int;
+#elif defined(LINUX) && defined(RISCV64)
+    //TODO
```

Comment:
I think an implementation is needed here before this should be merged.

Reply:
Removed the placeholder here to avoid duplication as the code for `tsc` was added recently by the external contributor working on OMR JIT.

I verified the added code with `clock_gettime `on Fedora_riscv and it works as expected.
```
gettimebase.c
#include <time.h>
#include <stdint.h>
#include <stdio.h>

#define J9CONST_U64(x) x##ULL
#define J9TIME_NANOSECONDS_PER_SECOND   J9CONST_U64(1000000000)

int main() {
uint64_t tsc = 0;
        struct timespec ts;
        if (0 == clock_gettime(CLOCK_MONOTONIC, &ts)) {
                tsc = ((uint64_t)ts.tv_sec * J9TIME_NANOSECONDS_PER_SECOND) + (uint64_t)ts.tv_nsec;
        }
  printf("\ntsc = %lu", tsc);
}

[jincheng@stage4]$ ./gettimebase
tsc = 1229793025165800
```